notebooks/MAST/TESS/interm_tesscut_astroquery/interm_tesscut_astroquery.ipynb

```diff
@@ -0,0 +1,507 @@
+    "  [Tesscut to Perform FFI Cutout](#tesscut_ID) <br>\n",
+    "  [Create Light Curve](#lightcurve) <br>\n",
+    "  [Additional Resources](#resources_ID) <br> \n",
+    "  [About this Notebook](#about_ID) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import Statements\n",
+    "<a id=\"imports_ID\"></a>\n",
+    "\n",
+    "We start with a few import statements.\n",
+    "- *numpy* to handle array functions\n",
+    "- *astropy.io fits* for accessing FITS files\n",
+    "- *astropy.wcs WCS* to interpret the World Coordinate Systems\n",
+    "- *matplotlib.pyplot* for plotting the data\n",
+    "- *astroquery.mast* for the catalogs and for tesscut.\n",
+    "\n",
+    "For ease of use later on, we also set the root url for the Tesscut package here.\n",
+    "\n",
+    "If your version of astroquery doesn't have tesscut, try\n",
+    "pip install astroquery==0.3.9.dev5086"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from astroquery.mast import Catalogs\n",
+    "from astroquery.mast import Tesscut\n",
+    "from astropy.coordinates import SkyCoord\n",
+    "from astropy.wcs import WCS\n",
+    "from astropy.io import fits\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "###CHANGE THE NEXT LINE in Pull Request ###\n",
+    "#This is the url we are using.\n",
+    "Tesscut._TESSCUT_URL = 'https://mastdev.stsci.edu/tesscut/api/v0.1/'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# DELETE THIS CELL\n",
+    "\n",
+    "#Next line puts your mast api token in a useful variable for later.\n",
+    "#See https://auth.mast.stsci.edu/info\n",
+    "masttoken = %env MAST_DV_TOKEN\n",
+    "Tesscut._session.headers[\"Authorization\"] = \"token %s\" % masttoken"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"catalog_ID\"></a>\n",
+    "## Get Coordinates of the Target Using Astroquery Catalogs\n",
+    "\n",
+    "Here we use the astroquery.mast Catalogs query_criteria function to request the RA and Dec of the object with TIC ID = 261105201.   \n",
+    "<p>\n",
+    "Here we do a cone search using Catalogs.query_object on the TIC catalog. The advantage of doing this (instead of a directly criteria query on the Tess Input Catalog) is that it gives us the nearby stars as well as the star we are looking for. The resulting table is sorted by distance from the requested object. We print out the ID and a few other TIC quantities to ensure we found the star we were looking for. "
+   ]
```

Comment:
Put Catalogs.query_object in backticks (\`Catalogs.query_object\`) to set off more?
"directly criteria query" -> "direct criteria query"